tensorflow/core/profiler/internal/gpu/rocm_tracer.cc

```diff
@@ -0,0 +1,1132 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "tensorflow/core/profiler/internal/gpu/rocm_tracer.h"
+
+#include <chrono>
+#include <iostream>
```

Comment:
```suggestion

```
I think iostream is not needed. Probably left behind from debugging.

Reply:
removed in the latest commit